dot-net-sdk/eppo-sdk.csproj

```diff
@@ -9,6 +9,7 @@
     <PackageProjectUrl>https://github.com/Eppo-exp/dot-net-server-sdk</PackageProjectUrl>
     <RepositoryUrl>https://github.com/Eppo-exp/dot-net-server-sdk</RepositoryUrl>
     <PackageId>Eppo.Sdk</PackageId>
+    <Version>1.0.0</Version>

```

Comment:
@agamkrbit Is this how versions are set in .net library projects? https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props

Reply:
Bumped to 2.0.0 as it seems the current version is `1.0.0` by default https://www.nuget.org/packages/Eppo.Sdk